src/tests/functional/inference_engine/ir_serialization/models/few_inputs.prototxt

```diff
@@ -0,0 +1,89 @@
+ir_version: 8
```

Comment:
It is not an IR format... maybe it should be a part of onnx tests in this case. Or use IR or Model as input.

Reply:
Replaced with ov::Model generated in runtime